src/main/java/se/sundsvall/invoicesender/util/Constants.java

```diff
@@ -0,0 +1,35 @@
+package se.sundsvall.invoicesender.util;
+
+import static java.util.function.Predicate.not;
+import static se.sundsvall.invoicesender.integration.db.entity.ItemStatus.IGNORED;
+import static se.sundsvall.invoicesender.integration.db.entity.ItemStatus.METADATA_INCOMPLETE;
+import static se.sundsvall.invoicesender.integration.db.entity.ItemStatus.NOT_SENT;
+import static se.sundsvall.invoicesender.integration.db.entity.ItemStatus.RECIPIENT_LEGAL_ID_NOT_FOUND_OR_INVALID;
+import static se.sundsvall.invoicesender.integration.db.entity.ItemStatus.RECIPIENT_PARTY_ID_NOT_FOUND;
+import static se.sundsvall.invoicesender.integration.db.entity.ItemType.INVOICE;
+
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+import se.sundsvall.invoicesender.integration.db.entity.ItemEntity;
+
+public final class Constants {
+
+	private Constants() {
+
+	}
+
+	public static final Predicate<ItemEntity> ITEM_IS_A_PDF = item -> item.getFilename().toLowerCase().endsWith(".pdf");
```

Comment:
Personal taste, but I would have left these predicates in the `ItemEntity` class